content/blog/2019-12-03-mark-padgham.md

```diff
@@ -0,0 +1,76 @@
+
+It’s in my own interest to continue to contribute to the community because it’s only in that way that what I have contributed continues to maintain its value. Without ongoing contributions to rOpenSci that whole thing stops at the point at which I left it and all I’ve got is to sit back and hope something happens. That’s worth nothing in comparison to me continuing to actively strive to foster, cultivate, grow, expand and enrich this ecosystem of tools in our little niche.
+
+
+### What will you be working on and what do you think are some of the key challenges?
+
+I’ll be working on developing robust tools to assess packages. As an inveterate coder, I love nothing more than to get up in the morning and find myself still sitting there at 4:00 in the afternoon in my “coding-hole”. But a big part of the initial stuff, which I’m really excited about, is about connecting various people who will ultimately be involved in guiding the expansion of rOpenSci into the statistical methods side of things. So it’s much more people-centered than the typical life of a 100% coder. Like so many rOpenSci projects, this aims to be very community-driven.
+
+I’ll really enjoy being between two worlds trying to think in general terms about developing this whole new endeavour [statistical software peer review] and ensuring the development is as general and generalizable as possible. While on the other side, having to be involved with profoundly important brilliant academic experts in various aspects of statistics & statistical software so it will be both very niche and very general at the same time. One of the key challenges will be working with experts in the statistical and software communities and having to foster consensus of opinion on their part in realms that have basically never been tried before. And on the other side, I’ve got a lot of concrete visions for extending testing well beyond anything that’s been done in a computational or quantitative way. Convincing leaders in statistics and statistical software that’s a good idea is certainly going to be challenging and very rewarding.
+
+
+### What would you like to see in the statistical software standards or package review landscape in two years?
+
+Actual testing of the statistical routines themselves. To date, all systems for evaluating software rely on signature keys of what reliable software might look like externally. It’s like it’s packaged up in a box and you’re evaluated by examining all the things on the outside of the box because getting inside the box is almost impossible. But R is also quite unique in that way, that it allows you to standardize things enough that you know what to look for and then pry into that.
```

Comment:
It’s like it’s packaged up in a box and you~~re evaluated by~~ can only examine~~ing all~~ the things on the outside of the box ...  But the R package system is also quite unique in that way,